Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/InputAxesChecker/DisplayInputResult.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Displays a specified axis / button value on a specific TextMesh.
+    /// Will display all active axes and buttons if the input type is None.
+    /// </summary>
+    public class DisplayInputResult : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Used for displaying data from input.")]
+        private TextMesh displayTextMesh = null;
+
+        [SerializeField]
+        [Tooltip("The type of input to read. Will read all active if set to None.")]
+        private AxisType inputType = AxisType.None;
+
+        [SerializeField]
+        [Tooltip("The axis number to read.")]
+        [Range(1, 28)]
+        private int axisNumber = 1;
+
+        [SerializeField]
+        [Tooltip("The button number to read.")]
+        [Range(0, 19)]
+        private int buttonNumber = 0;
+
+        private void OnValidate()
+        {
+            switch (inputType)
```

Comment:
Any chance we could log a warning if the range of either thing doesn't match the axis/button counts of ControllerMappingLibrary.cs? Ideally if someone is using this they should be set up for success (i.e. be told that hey this hasn't been updated with the new axis values)

Reply:
Since the number of axes and buttons is explicitly set by Unity, I think we're okay here.

![image](https://user-images.githubusercontent.com/3580640/64553278-f02bdf80-d2ed-11e9-8c50-85979d723089.png)

I do like the idea of logging a warning anyway, but since `ControllerMappingLibrary` is an editor script and this is a runtime script, it was less straight forward.